k8s/prod-redis-server.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
A quick one, what is the main difference between `prod` and `stage` for Redis?
Is it the NodePort? Or the image url/version?

Reply:
For now, it's just the `k8s namespace`. Both staging and production are being served using the same redis image